### PR TITLE
OSDOCS#6512: Update the callout decriptions

### PR DESCRIPTION
**Pull request by xenolinux**
Version(s): 4.13 only

Issue: https://issues.redhat.com/browse/OSDOCS-6512

Link to docs preview: https://61215--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_azure/installing-azure-customizations.html#installing-azure-user-defined-tags_installing-azure-customizations

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by jinyunma** [comment]
the update looks good to me, but I could not open the doc review link.
@xenolinux could you help to check?

**Comment by xenolinux** [comment]
> the update looks good to me, but I could not open the doc review link. @xenolinux could you help to check?

@jinyunma Done. Fixed the preview link

**Comment by jinyunma** [comment]
Thanks @xenolinux 
lgtm

File: modules/installation-user-defined-tags-azure.adoc [hunk 1]
```diff
@@ -28,44 +28,47 @@ compute: <4>
...
-<8> Specifies the resource group for the base domain of the Azure DNS zone.
-<9> Specifies the name of the Azure cloud environment. You can use the `cloudName` field to configure the Azure SDK with the Azure API endpoints. If you do not provide value, the default value is Azure Public Cloud.
-<10> Defines the additional keys and values that the installer adds as tags to all Azure resources that it creates.
+<6> The configuration for the machines that comprise the control plane. The `controlPlane` section is a single mapping. The first line of the `controlPlane` section must not begin with a hyphen, `-`. You can use only one control plane pool. If you do not provide these parameters and values, the installation program provides the default value.
+<7> To enable or disable simultaneous multithreading, or `hyperthreading`. By default, simultaneous multithreading is enabled to increase the performance of your machines' cores. You can disable it by setting the parameter value to `Disabled`. If you disable simultaneous multithreading in some cluster machines, you must disable it in all cluster machines.
```
Comment (snarayan-redhat) [inline modules/installation-user-defined-tags-azure.adoc:65]: `If you disable simultaneous multithreading in some cluster machines, you must disable it in all cluster machines.`
Does this mean you cannot disable simultaneous multithreading in just in selected cluster machines? If yes, you can mention that.

**Review by snarayan-redhat — COMMENTED** [review]
Few suggestions and one question.

File: modules/installation-user-defined-tags-azure.adoc [hunk 2]
```diff
@@ -28,44 +28,47 @@ compute: <4>
...
-<7> The cluster network plugin to install. The supported values are `OVNKubernetes` and `OpenShiftSDN`. The default value is `OVNKubernetes`.
-<8> Specifies the resource group for the base domain of the Azure DNS zone.
-<9> Specifies the name of the Azure cloud environment. You can use the `cloudName` field to configure the Azure SDK with the Azure API endpoints. If you do not provide value, the default value is Azure Public Cloud.
-<10> Defines the additional keys and values that the installer adds as tags to all Azure resources that it creates.
+<6> The configuration for the machines that comprise the control plane. The `controlPlane` section is a single mapping. The first line of the `controlPlane` section must not begin with a hyphen, `-`. You can use only one control plane pool. If you do not provide these parameters and values, the installation program provides the default value.
```
[the remark this answers is not in the document]
Comment (xenolinux) [reply modules/installation-user-defined-tags-azure.adoc:64]: @jinyunma Would you please help with the default value[1] for callout <6>? 
[1] `If you do not provide these parameters and values, the installation program provides the default value.`

[the remark this answers is not in the document]
Comment (jinyunma) [reply modules/installation-user-defined-tags-azure.adoc:64]: ControlPlane object contains several sub parameters as below
```
$ ./openshift-install explain installconfig.controlPlane
KIND:     InstallConfig
VERSION:  v1

RESOURCE: <object>
  ControlPlane is the configuration for the machines that comprise the control plane.

FIELDS:
    architecture <string>
      Default: "amd64"
      Valid Values: "","amd64"
      Architecture is the instruction set architecture of the machine pool. Defaults to amd64.

    hyperthreading <string>
      Default: "Enabled"
      Valid Values: "","Enabled","Disabled"
      Hyperthreading determines the mode of hyperthreading that machines in the pool will utilize. Default is for hyperthreading to be enabled.

    name <string> -required-
      Name is the name of the machine pool. For the control plane machine pool, the name will always be "master". For the compute machine pools, the only valid name is "worker".

    platform <object> -required-
      Platform is configuration for machine pool specific to the platform.

    replicas <integer>
      Format: int64
      Replicas is the machine count for the machine pool.

```
If user don't set ControlePlane object explicitly in install-config, indeed those sub parameters will be set to default value by installer, do we need to describe each parameter's default value here? I see same statements are shown in installation doc on other platforms, for example, aws doc https://docs.openshift.com/container-platform/4.13/installing/installing_aws/installing-aws-customizations.html.

[the remark this answers is not in the document]
Comment (xenolinux) [reply modules/installation-user-defined-tags-azure.adoc:64]: Yes. Other instances are referring to the default value only. Also, mentioning the whole default params list would be the overload here IMO. Hence keeping it as is.

**Pull request merged**